### PR TITLE
Adding switch to allow for filing work items for results including ba…

### PR DESCRIPTION
**Pull request by cfaucon**
…selinestate unchanged.

### CODE REVIEW

File: src/Sarif.WorkItems/SarifWorkItemsExtensions.cs [hunk 1]
```diff
@@ -46,7 +50,7 @@ public static string GetFingerprintSplittingStrategyId(this Result result)
             return $"{result.GetProperty("OrganizationName")}:{result.GetProperty("EtlEntity")}:{result.PartialFingerprints["SecretHash/v1"]}";
         }
 
-        public static string CreateWorkItemTitle(this Run run)
+        public static string CreateWorkItemTitle(this Run run, bool shouldFileUnchanged = false)
```
Comment (rtaket) [inline src/Sarif.WorkItems/SarifWorkItemsExtensions.cs:53]: >false [](start = 90, length = 5)

Why default the value here?

File: src/Sarif.WorkItems/SarifWorkItemsExtensions.cs [hunk 2]
```diff
@@ -121,17 +125,17 @@ public static List<string> GetToolNames(this SarifLog log)
                 .ToList();
         }
 
-        public static int GetAggregateFilableResultsCount(this SarifLog log)
+        public static int GetAggregateFilableResultsCount(this SarifLog log, bool shouldFileUnchanged = false)
```
Comment (rtaket) [inline src/Sarif.WorkItems/SarifWorkItemsExtensions.cs:128]: >false [](start = 104, length = 5)

why default?

File: src/Sarif.WorkItems/SarifWorkItemsExtensions.cs (hunk 1 above)
Comment (cfaucon) [reply src/Sarif.WorkItems/SarifWorkItemsExtensions.cs]: removed default

File: src/Sarif.WorkItems/SarifWorkItemsExtensions.cs (hunk 2 above)
Comment (cfaucon) [reply src/Sarif.WorkItems/SarifWorkItemsExtensions.cs]: removed default

**Pull request merged**